tests/imposter/scripts/get-question.groovy

```diff
@@ -0,0 +1,23 @@
+def serializer = new groovy.json.JsonOutput();
```

Comment:
Based on the comments from earlier, it would be interesting to see a branch on this that uses full Java instead of Groovy.

(I'm still sad that JavaScript turned out to be a janky failure :sob:)